source/posts.yaml

```diff
@@ -311,3 +311,41 @@
           schema:
             $ref: "#/definitions/PostList"
 
+  '/teams/{team_id}/channels/{channel_id}/posts/{post_id}/copy':
```

Comment:
Do we really need another API for this? Why not just create a new post in the other channel?